config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  mount Ckeditor::Engine => '/ckeditor'
```

Comment:
chỗ này anh hỏi tí.
url chuyển sang có thêm /ckeditor khi nào?

Reply:
cái này nó tự migrate ra khi e install gem ckeditor..cái này không phải là chuyển sang /ckeditor mà nó. mà kiểu như là khi rails hoạt động thì nó nó sẽ dùng công cụ ckeditor khi mình sử dụng nó trong view ạ ..e mò mẫm thôi ..a có thể đọc chích xác ở đây :)) https://guides.rubyonrails.org/engines.html#mounting-the-engine